src/com/madalinb/coffeeshop/data/CoffeeShop.kt

```diff
@@ -0,0 +1,7 @@
+package com.madalinb.coffeeshop.data
+
+data class CoffeeShop(var name: String, var x: Double, var y: Double) {
```

Comment:
Usually data classes are meant to store only data and no functions, can you think of why is the case?

Reply:
Refactored the class.